frontend/src/Select.js

```diff
@@ -80,4 +79,49 @@ function OrderingPrioritySelect(props) {
   );
 }
 
-export { Select as default, SubassemblySelect, OrderingPrioritySelect };
+function OrderingStatusSelect(props) {
+  const value = props.value ? parseType("Status", props.value._type) : "";
+
+  // Option: style
+  const statuses = {
```

Comment:
Nit: We should probably refactor these style dictionaries out eventually, but not important now.

Reply:
Agreed and agreed.